frontend/mr-admin-flate/src/pages/enhet/EnhetsoversiktPage.tsx

```diff
@@ -14,7 +14,7 @@ import styles from "../tiltaksgjennomforinger/Oversikt.module.scss";
 export function EnhetsoversiktPage() {
   const { data: ansattData } = useHentAnsatt();
   const { data, isFetching, isError } = useTiltaksgjennomforingerByEnhet(
-    ansattData?.hovedenhet
+    ansattData?.hovedenhet ?? ""
```

Comment:
Er ikke storfornøyd med denne, hva blir beste å sørge for at vi kun henter gjennomføringerByEnhet _etter_ at vi faktisk har hentet enheten?

Sende ansattData/enhet via props i stedet?

Reply:
`useTiltaksgjennomforingerByEnhet` har en `enabled: !!enhet` i seg som gjør at kallet bare går dersom enhet er en truthy verdi, så jeg tenker denne er ok. Kunne eventuelt hatt `undefined` istedenfor tom streng

Reply:
> useTiltaksgjennomforingerByEnhet har en enabled: !!enhet i seg som gjør at kallet bare går dersom enhet er en truthy verdi 

Se der ja, tror ikke visste om denne! :smile: 

Etter at jeg gravde litt mer i typene så ser jeg at `hovedenhet` i utgangspunktet ikke er nullable, så ved å skru litt om på typer og props så kunne dette gjøres helt typesafe likevel :)